jupyterlab_github/__init__.py

```diff
@@ -14,47 +14,56 @@
 
 
 link_regex = re.compile(r'<([^>]*)>;\s*rel="([\w]*)\"')
-GITHUB_API = 'https://api.github.com'
+
 
 class GitHubConfig(Configurable):
     """
-    A Configurable that declares the 'client_id' and 'client_secret'
-    parameters.
+    A Configurable that declares the `access_token` parameter.
     """
-    client_id = Unicode('', config=True,
```

Comment:
I think that an access token is probably better, but I'm uncomfortable removing support for the oauth app approach. Can we support both, and have them toggle-able using another config option?

Reply:
Yep, certainly possible - I'll look into making those changes...